.evergreen/config.yml

```diff
@@ -3111,17 +3111,28 @@ buildvariants:
     - name: "aws-auth-test-5.0"
     - name: "aws-auth-test-latest"
 
-- matrix_name: "load-balancer"
+- matrix_name: "load-balancer-awslinux"
```

Comment:
Let's revert the name change to avoid loosing task history.

Reply:
Done